### PR TITLE
port/zephyr: Adding support for the nucleo_h743zi board.

### PR DESCRIPTION
**Pull request by embeddedSystemsWorld**

### CODE REVIEW

File: ports/zephyr/boards/nucleo_h743zi.conf [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# disable console subsys to get REPL working
+CONFIG_CONSOLE_SUBSYS=n
+
+# flash and file system config
+CONFIG_FLASH=y
+CONFIG_FLASH_MAP=y
+CONFIG_FLASH_PAGE_LAYOUT=y
+
+CONFIG_FILE_SYSTEM=y
+CONFIG_FILE_SYSTEM_LITTLEFS=y
```
Comment (MaureenHelm) [inline ports/zephyr/boards/nucleo_h743zi.conf:10]: This is probably not what you want, unless you intend to use LittleFS under the hood in C without exposing it to MicroPython scripts. Instead you need to make sure that Makefile options `MICROPY_VFS_LFS1` or `MICROPY_VFS_LFS2` are set. The latter is set by default in the Zephyr port. See https://github.com/micropython/micropython/blob/master/ports/zephyr/Makefile#L19-L21

Comment (embeddedSystemsWorld) [reply ports/zephyr/boards/nucleo_h743zi.conf]: That's right, I didn't really see that. Thanks for the input! I have adjusted the config file accordingly. Using the make process as following, the filesystem is working:

`make BOARD=nucleo_h743zi MICROPY_VFS_LFS2=1  flash`

**Comment by embeddedSystemsWorld** [comment]
Check commit message formatting / build (pull_request)  is failing. But i do not see any reason as there is a ': ' and a '.' within the message. Any Input?

**Comment by stinos** [comment]
There should be a capital after the `:` 
See https://github.com/micropython/micropython/pull/6821

File: ports/zephyr/boards/nucleo_h743zi.conf (hunk 1 above)
Comment (MaureenHelm) [reply ports/zephyr/boards/nucleo_h743zi.conf]: The net result looks fine, but I recommend that you squash your cleanup commit into your original commit

Comment (embeddedSystemsWorld) [reply ports/zephyr/boards/nucleo_h743zi.conf]: Good idea. Tnx!

**Comment by dpgeorge** [comment]
Thanks for this.  Rebased, squashed and merged in 6c4a5d185d7e70c22c0f72fb1cd8add251600548

**Pull request closed without merging**